great_expectations/experimental/datasource/fabric.py

```diff
@@ -216,7 +214,7 @@ class PowerBITable(_PowerBIAsset):
     fully_qualified_columns: bool = False
     num_rows: Optional[int] = None
     multiindex_hierarchies: bool = False
-    pandas_convert_dtypes: bool = True
+    mode: str = "xmla"
```

Comment:
If there are other we can use a `Literal["xlma", "foo", "bar"]` here and then users will get input validation if they enter something wrong.
It will also be part of the generated JSON Schemas which we use to power our UI.

Reply:
Can also define it as a module level constant and re-use it.

```python
Mode = Literal["xlma", "other_mode", "another"]



@public_api
class PowerBIMeasure(_PowerBIAsset):
    """Microsoft PowerBI Measure."""
    _reader_method: ClassVar[FabricReaderMethods] = "evaluate_measure"
    type: Literal["powerbi_measure"] = "powerbi_measure"
    ...
    multiindex_hierarchies: bool = False
    mode: Mode = "xmla"
```